src/ts/state/reducer.ts

```diff
@@ -76,8 +83,8 @@ export interface IBlockInFlight {
  * Handy when, for example, consumers know that parts of the document are
  * exempt from checks.
  */
-export type IIgnoreMatch = (match: IMatch) => boolean;
-export const includeAllMatches: IIgnoreMatch = () => false;
+export type IIgnoreMatchPredicate = (match: IMatch) => boolean;
```

Comment:
Another type starting with `I`, albeit one you renamed rather than introduced...

Reply:
😅 oh god, the sooner I get rid of these the better ...